definitions/infra-prometheus-redis/definition.yml

```diff
@@ -0,0 +1,22 @@
+domain: infra
```

Comment:
domain and type are meant to be uppercase (we should document this better!)

Reply:
done